### ISSUE TITLE
Multitask doesn't respect $SHELL

### ISSUE THREAD

**Issue by calebmeyer** [leakec/multitask#1]
I'm on macOS, using iTerm2 as my terminal, and the Friendly Interactive Shell (fish) as my shell. I have fish all set up with the correct environment variables, functions, aliases, etc.

When I run this plugin, all commands are executed in bash:
<img width="572" alt="image" src="https://github.com/imsnif/multitask/assets/3342777/a376e87d-2386-4db4-8672-0a7768568e76">

I can get around this with `fish -c "api; cargo run"` or similar, but it would be nice if the default shell was used like the default $EDITOR.

**Comment by imsnif** [comment]
Yes, I am with you.

I'll try to implement the API on the Zellij side to allow this when I get the chance.

**Comment by leakec** [comment]
@imsnif Just added a pull request (#2) that I believe solves this. zellij `0.38` allows a configuration to be set for the plugin. Users can pass in `shell=$SHELL` when invoking the plugin, which is stored on the state. This is passed to the commands that are run in place of what used to be `bash`. Note, `bash` is still used as the default if `shell` is not set in the configuration when loading the plugin.

**Comment by leakec** [comment]
@calebmeyer This has been implemented in the 0.38.2 release. Invoking the plugin using
```
zellij action start-or-reload-plugin file:~/zellij-plugins/multitask.wasm --configuration "shell=$SHELL"
```
should run commands with your default shell rather than bash.
